python/lsst/meas/algorithms/loadReferenceObjects.py

```diff
@@ -306,15 +306,15 @@ def makeMinimalSchema(filterNameList, addFluxSigma=False,
                 field = "%s_flux" % (filterName,),
                 type = numpy.float64,
                 doc = "flux in filter %s" % (filterName,),
-                units = "?",
+                units = "count",
             )
         if addFluxSigma:
             for filterName in filterNameList:
                 schema.addField(
                     field = "%s_fluxSigma" % (filterName,),
                     type = numpy.float64,
                     doc = "flux uncertainty in filter %s" % (filterName,),
-                    units = "?",
+                    units = "count",
```

Comment:
All your options are listed in http://www.ivoa.net/documents/VOUnits/20140523/VOUnits-REC-1.0-20140523.pdf